### ISSUE TITLE
errors during execution

### ISSUE THREAD

**Issue by quotschmacher** [Pittini/iobroker-Batterienauswertung#6]
i added rooms and functions to some devices of the zigbee-adapter, enanled logging and telegram in the configuration and during execution got:

```

javascript.0 | 2020-12-18 23:45:16.588 | error | (16424) at runMicrotasks (<anonymous>)
-- | -- | -- | --
javascript.0 | 2020-12-18 23:45:16.588 | error | (16424) at /opt/iobroker/node_modules/standard-as-callback/built/index.js:19:49
javascript.0 | 2020-12-18 23:45:16.587 | error | (16424) at tryCatcher (/opt/iobroker/node_modules/standard-as-callback/built/utils.js:11:23)
javascript.0 | 2020-12-18 23:45:16.587 | error | (16424) at /opt/iobroker/node_modules/iobroker.objects-redis/index.js:1:93703
javascript.0 | 2020-12-18 23:45:16.586 | error | (16424) at /opt/iobroker/node_modules/iobroker.js-controller/lib/adapter.js:2870:21
javascript.0 | 2020-12-18 23:45:16.586 | error | (16424) at /opt/iobroker/node_modules/iobroker.javascript/lib/sandbox.js:1943:38
javascript.0 | 2020-12-18 23:45:16.585 | error | (16424) at Object.<anonymous> (script.js.common.BatterieWaechter:94:13)
javascript.0 | 2020-12-18 23:45:16.585 | error | (16424) at main (script.js.common.BatterieWaechter:218:5)
javascript.0 | 2020-12-18 23:45:16.584 | error | (16424) at CheckDeadBatt (script.js.common.BatterieWaechter:271:5)
javascript.0 | 2020-12-18 23:45:16.584 | error | (16424) at MakeTable (script.js.common.BatterieWaechter:487:81)
javascript.0 | 2020-12-18 23:45:16.582 | error | (16424) Error in callback: TypeError: Cannot read property 'toFixed' of undefined

```

can i give you any help resolving this problem?

**Comment by quotschmacher** [comment]
i think the device that causes the error is a dimmer that i paired directly to a lamp. it is still listed in my iobroker objects but doesn't show any value at the battery state. maybe you should catch tose cases where no value is presented

**Comment by Pittini** [comment]
fixed with 1.7.0
